### PR TITLE
[SKIP CI] Document plugin and ESX service version compatibility

### PR DESCRIPTION
**Pull request by ashahi1**
This PR addresses the issue #2025

### CODE REVIEW

File: docs/external/faq.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ Edge release: A monthly release with new features and bug fixes.
 
 Stable release: A quarterly release with reliable/stable updates.
 
+***Note: From 0.19 release of vDVS, the plugin has to have a ESX service thats at least 0.19 or later. This applies for plugins on Linux platform.***
```
Comment (shuklanirdesh82) [inline docs/external/faq.md]: Let's not add here and move out as a separate FAQ entry for any user to search error message to get quick answer.

Comment (ashahi1) [reply docs/external/faq.md]: Done

File: docs/external/faq.md [hunk 2]
```diff
@@ -88,6 +88,9 @@ e.g.
 systemctl restart docker
 ```
 
+#### I'm not able to create volume and I see "VolumeDriver.Create: Device not found" error
+
+From 0.19 release of vDVS, the plugin has to have a ESX service thats at least 0.19 or later. This applies for plugins on Linux platform. [#2023](https://github.com/vmware/docker-volume-vsphere/issues/2023)
```
Comment (shuklanirdesh82) [inline docs/external/faq.md]: `ESX service thats at least 0.19` => `ESX service at least 0.19` or `thats` -> `that's` What do you mean by `thats`?

Comment (shuklanirdesh82) [inline docs/external/faq.md]: Let's not mention `Linux platform` explicitly, it should be OS independent.

/CC @govint  to confirm

Comment (govint) [reply docs/external/faq.md]: Could mention VIB (use VIB name with the version) perhaps vs. ESX service. 

For Windows the device identification in the plugin is the same as it was, while for Linux the device identification has been upgraded to detect devices on secondary PCI buses.

Comment (govint) [reply docs/external/faq.md]: This issue will happen only with the plugin on Linux, but agree there is no need for the windows plugin to use an older VIB version. Its best that the users always upgrade both the pluigin and the VIB to matching levels.

**Pull request merged**